src/env.ts

```diff
@@ -45,6 +46,7 @@ export const defaultEnv: Env = {
   GQL_API_URL: 'http://localhost:8080/v1/graphql',
   GQL_API_WS_URL: 'ws://localhost:8080/v1/graphql',
   HASURA_GRAPHQL_JWT_SECRET: '',
+  HASURA_METADATA_API_URL: 'http://hasura:8080/v1/metadata',
```

Comment:
Yeah, I tried using the localhost connection initially, but it could only connect over `hasura`.

Reply:
Ahh I see, we only actualy use `GQL_API_URL` for the playground setup, which then has the browser making the actual GQL requests, so localhost makes sense. In this case, `node` within the gateway container is making the request, so it needs the docker network DNS entry instead of `localhost`